sdk/deviceupdate/Azure.ResourceManager.DeviceUpdate/src/autorest.md

```diff
@@ -64,4 +64,10 @@ directive:
           "type": "string"
         }
       }
+  - from: deviceupdate.json
+    where: $.definitions
+    transform: >
+      $.Location['x-ms-client-name'] = 'DeviceUpdateAccountLocationDetail';
+      $.Location.properties.role['x-ms-enum'].name = 'DeviceUpdateAccountLocationRole';
+      $.Account.properties.properties.properties.sku['x-ms-enum'].name = 'Sku';
```

Comment:
We don't edit these files, we use this autorest.md file to modify these names during the process of the generation. I renamed them using the configuration here and committed the change to your PR. You can take a look, if you're not ok with the names that I changed, feel free to rename them, just need to change the configuration.